renku/domain_model/template.py

```diff
@@ -530,7 +531,8 @@ def from_project(cls, project: Optional["Project"]) -> "TemplateMetadata":
         # NOTE: Always set __renku_version__ to the value read from the Dockerfile (if available) since setting/updating
         # the template doesn't change project's metadata version and shouldn't update the Renku version either
         renku_version = metadata.get("__renku_version__")
-        metadata["__renku_version__"] = str(read_renku_version_from_dockerfile() or renku_version or __version__)
```

Comment:
All of these values can be `None` and if we use `str` on the result we'll get `"None"`.

Reply:
before it was `str(read_renku_version_from_dockerfile()) or renku_version or __version__` and that led exactly to that error